bblfsh/pyuast.c

```diff
@@ -0,0 +1,119 @@
+#include "uast.h"
```

Comment:
Is there a code style for python modules? If not, I guess we should follow Google C++ code style, which is not been followed here.

Reply:
Not that I know for - or resulted from a quick search. Since @manucorporat probably won't be working on this anymore I'll change it soon.